backend/common/src/main/java/ai/verta/modeldb/common/CommonConstants.java

```diff
@@ -17,6 +17,12 @@ public interface CommonConstants {
   String LIQUIBASE_MIGRATION = "LIQUIBASE_MIGRATION";
   String RUN_LIQUIBASE_SEPARATE = "RUN_LIQUIBASE_SEPARATE";
   String BACKEND_PID = "verta-backend.pid";
+  String FILENAME = "FileName";
+  String PUT = "put";
+  String GET = "get";
```

Comment:
very strongly recommend against putting simple words like this into a constants file. Just use them as-is, in-line.

Reply:
this is used across multiple backend so added it in CommonConstants.java
It is used multiple places so as per the sonarqube suggestions i have extracted it in constant file.

Reply:
we should adjust our sonarqube settings to not flag this, then. Putting any arbitrary duplicated constant in a separate file is silly and just makes the code more confusing.

Reply:
Fixed